sdks/java/testing/jpms-tests/build.gradle

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+  id 'org.apache.beam.module'
+}
+javaVersion="1.11"
+applyJavaNature(
+  enableChecker:false,
+  exportJavadoc: false,
+  publish: false,
+  disableLintWarnings: ['requires-transitive-automatic', 'requires-automatic']
+)
+provideIntegrationTestingDependencies()
+enableJavaPerformanceTesting()
+
+description = "Apache Beam :: SDKs :: Java :: Testing :: JPMS Tests"
+ext.summary = "E2E test for Java 9 modules"
+
+dependencies {
+  compile project(path: ":sdks:java:core", configuration: "shadow")
+  compile project(path: ":runners:direct-java")
```

Comment:
I believe this would be `runtime` or potentially `testRuntimeOnly`.

If I understand this one correctly right now, it runs the cloned WordCount on the DirectRunner. You may want to do some of the hacking done [here](https://github.com/apache/beam/blob/38403698934c422968320a0bdd834e8b9ae598a3/examples/java/build.gradle#L99) to run it on a variety of runners. I don't know which might (1) be able to run it or (2) have infrastructures that would make this test check anything meaningful.

Reply:
OK actually tidying this would be a good follow-up. I don't want to hold up the hold change on this, though.